_posts/2020-09-27-october-meetup.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title:  "October 2020"
+date:   2020-09-27 00:00:00
+categories: events
+---
+
+[CocoaHeads October 2020]
+
+**When?**
+
+Thursday, 01.10.2020, 19:00 onwards
+
+**Where?**
+
+REMOTE!
+
+We're continuing to keep Cocoaheads online for this month as well 🎉
+We're using Jitsi, this time around as well. 
+
+*(once your RSVP for the [Meetup event](https://www.meetup.com/CocoaHeads-Hamburg/events/jgthtrybclbjb/) you'll be able to see the link in the event location sidebar on Meetup)*
+
+⏰ Our schedule ⏰
+
+As a result from the poll we shared with you last time we'll have a lighter, easier to follow (remotely) version of the CocoaHeads this time, with just 1 talk and the socialising part.
+
+Feel free to join us via Slack, where we'll keep you up-to-date on any immediate announcements.
+👉🏼 [Click here](https://slack.cocoaheads.hamburg) to join our workspace 'cocoaheadshamburg.slack.com'
+
+19:20 - You can start joining the Video Call
+
+19:30 - Meeting starts 🎉
+
+19:45 - 1️⃣
+Alexander: Silicon Soul: The Machine we must not talk about
```

Comment:
```suggestion
Alexander von Below: Silicon Soul: The Machine we must not talk about
```
Let's put his full name here, as I think many people will only recognize him by his last name. Once we published it, we can double-check with Alex whether he wants us to put it somehow differently.

Reply:
Sure. I wasn't sure of this being his Full Name, thanks for confirming!